Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,52 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
+﻿using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Lesson3.Demo();
+        }
+
+        public static void Descriptor()
+        {
+            //ask user for their name, age, weight, and height
+            string name = PromptString("Name: ");
+            string surname = PromptString("Surname: ");
+            int age = PromptInt("Age: ");
+            float weight = PromptFloat("Weight: ");
+            float height = PromptFloat("Height: ");
+            
+            //print a sentence about user
+            Console.WriteLine($"{name} {surname} is {age} years old, their weight is {weight}Kg and their height is {height}cm");
+
+            //calculate and print the BMI of the user
+            float bmi = CalculateBmi(weight, height / 100);
+            Console.WriteLine($"They have a BMI of {bmi}");
+        }
```

Comment:
That's a sneaky way to keep you code where it was. The task - as far as I understood it - wanted from you that you move the actual logic over to the Demo() function, not only a reference to the logic.

Reply:
You mean in the Descriptor() function?

Reply:
Yeah.
Following tasks need to be done (as clarified by Kai moments ago)
> Move every functions except Main() to Lesson3.cs
> Change Program.X() to Lesson3.X() where needed.

Reply:
I moved all the functions to Lesson3 and explicitly made two different float variables, one for the height in cm and one for the height in m.